packages/gimbal/src/command/audit/program.ts

```diff
@@ -9,7 +9,7 @@ export default new Command({
     {
       defaultValue: 'build',
       description: 'Directory storing the build artifacts relative to the --cwd (defaults to "build")',
-      flag: '--build-dir [dir]',
+      flag: '--build-dir <dir>',
```

Comment:
This just means the `dir` value is required. Square brackets allow it to be optional if you pass the `--build-dir` flag.